contracts/Makefile

```diff
@@ -20,7 +20,7 @@ upgrade-gw-diamond:
 	./ops/upgrade-gw-diamond.sh $(NETWORK)
 
 upgrade-sa-diamond:
-	./ops/upgrade-sa-diamond.sh $(NETWORK)
+	./ops/upgrade-sa-diamond.sh $(NETWORK) $(SUBNET_ACTOR_ADDRESS)
 
 upgrade-sr-diamond:
```

Comment:
Hi @snissn, just a quick question for upgrading `subnet-registry`, if the subnet actor code has updated, and an we want the registry to deploy new version of `SubnetActor`, after an upgrade of SubnetRegistry, will the registry start deploying the newer version of the Subnet Actor?

Reply:
Hey will! The answer should be yes. Using the upgrade code to upgrade `src/subnetregistry/RegisterSubnetFacet.sol` in the registry should change the deploy path for new subnets.